Design doc for bring-your-own-host (#901)
vmware-tanzu/tanzu-framework: docs/design/bringyourownhost-design.md

> # Bring your own host enhancement design

Reviewer: ```suggestion
# Bring-Your-Own-Host Design
```

## Abstract

> The Bring Your Own Host aka BYOH aims at enabling Kubernetes cluster life cycle management on user provided hosts (most of the case are bare metal hosts).

Reviewer: ```suggestion
The Bring Your Own Host aka BYOH aims at enabling Kubernetes cluster life cycle management on user-provided hosts (bare metal hosts in most cases).
```

> But currently only 1 infrastructure provider is supported in tanzu-framework,  this is to enhance current tanzu-framework so that users can use tanzu CLI to create BYOH workload cluster.

Reviewer: I am not sure I understand this. Can you clarify?
